near-plugins-derive/src/pausable.rs

```diff
@@ -190,7 +190,10 @@ pub fn if_paused(attrs: TokenStream, item: TokenStream) -> TokenStream {
         let mut __check_paused = true;
         #bypass_condition
         if __check_paused {
-            ::near_sdk::require!(self.pa_is_paused(#fn_name.to_string()), "Pausable: Method must be paused");
+            ::near_sdk::require!(
+                self.pa_is_paused(#fn_name.to_string()),
+                "Pausable: The feature corresponding to this escape hatch is not paused"
```

Comment:
Initially I thought that would cost extra gas since it requires something like `format!` and didn’t do it for that reason. Your comment made me take another look: since it is inside `quote!`, the function name can be filled in at compile time. Changed in 2c00f34.